From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Fiona Ebner <f.ebner@proxmox.com>
Date: Thu, 27 Apr 2023 10:28:08 +0200
Subject: [PATCH] savevm-async: also initialize compression counters

Like is done in the migration code. While the compression migration
capability is not used by Proxmox VE currently, it's still worth to
be prepared for the future.

Signed-off-by: Fiona Ebner <f.ebner@proxmox.com>
---
 migration/savevm-async.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/migration/savevm-async.c b/migration/savevm-async.c
index de91506821..effe6d1e0d 100644
--- a/migration/savevm-async.c
+++ b/migration/savevm-async.c
@@ -394,6 +394,7 @@ void qmp_savevm_start(const char *statefile, Error **errp)
      */
     migrate_init(ms);
     memset(&ram_counters, 0, sizeof(ram_counters));
+    memset(&compression_counters, 0, sizeof(compression_counters));
     ms->to_dst_file = snap_state.file;
 
     error_setg(&snap_state.blocker, "block device is in use by savevm");
